website/src/app/kb/architecture/core-components/readme.mdx

```diff
@@ -0,0 +1,197 @@
+  Relays do not possess the ability to decrypt or otherwise tamper with the data
+  being relayed.
+</Alert>
+
+Relays are globally distributed across multiple regions to ensure low latency
+and reliable connectivity wherever your workforce happens to be.
+
+## Components you manage
+
+These are components you deploy and manage in your environment.
+
+### Clients
+
+The official Firezone Client applications are hosted from the following
+locations:
+
+- **macOS / iOS**:
+  [Apple App Store](https://apps.apple.com/us/app/firezone/id6443661826)
+- **Android**:
+  [Google Play Store](https://play.google.com/store/apps/details?id=dev.firezone.android)
+- **Windows**:
+  [GitHub releases](https://www.github.com/firezone/firezone/releases)
+- **Linux**:
+  [GitHub releases](https://www.github.com/firezone/firezone/releases)
+
+We recommend only using Clients from these official sources to ensure you're
+always running an authentic version with the latest security patches.
+
+These can be further distributed using your organization's preferred MDM
+solution, if applicable.
+
+### Gateways
+
+Gateways are Linux binaries that run on your infrastructure. They can be
+deployed as Docker containers, systemd services, or even as standalone
+executables onto a wide variety of Linux configurations.
+
+Gateways were designed to be portable, extremely lightweight, and require no
+external dependencies. This makes them easy to deploy and manage at scale.
+
+Gateways no persistent storage and only a handful of environment variables to be
+set in order to function. For more information on deploying Gateways, see the
+[Gateway deployment guide](/kb/deploy/gateways).
+
+Gateways are hosted from the following locations:
+
+- Binary: [GitHub releases](https://www.github.com/firezone/firezone/releases)
+- Docker: [GitHub Container Registry](ghcr.io/firezone/gateway)
+
+### Resources
+
+Resources are the applications and services you want to secure access to. These
+are defined in the admin portal as one of the following:
+
+- A CIDR range, such as `10.0.0.0/24`
+- An IP address, such as `172.16.55.1`
+- A DNS name with optional wildcard matching, such as `gitlab.company.com` or
+  `*.example.com`
+
+Resources can even be public SaaS applications. This can be used to secure
+access to public apps such as GitHub, Salesforce, or Slack using DNS names like
+`*.github.com`, `*.salesforce.com`, or `*.slack.com`.
+
+### Policies
+
+Policies are the rules that determine who can access which Resources. They're a
+simple one-to-one mapping of Group to Resource.
+
+By default, no Policies are created, meaning all access is denied by default and
+must be explicitly allowed by creating a Policy. For convenience, Firezone
+provides a special `Everyone` Group that can be applied to Policies to allow
+access to all authenticated users in your organization.
+
+Policies are evaluated in realtime by the [Policy Engine](#policy-engine) each
+time a Client attempts to connect to a Resource.
```

Comment:
Do we disconnect existing connections if a policy is updated that removes access to a given resource?

Reply:
Yes, the resource is removed from the routing table on the client, and enforced on the Gateway -- cc @conectado to confirm this.

Reply:
That's what I thought but wanted to make sure.  Does that mean policies are technically evaluated at the time of policy change as well as at connection time?

Reply:
Yeah there are two "entrypoints" where we can determine when connections are allowed:

- when requested by a client
- when re-evaluated from a configuration change

I'll update this part with that

Reply:
Done